index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css" integrity="sha512-1sCRPdkRXhBV2PBLUdRb4tMg1w2YPf37qatUFeS7zlBy7jJI8Lf4VHwWfZZfpXtYSLy85pkm9GaYVYMfw5BC1A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Front End Development</title>
+</head>
+<body>
+    <section class="container-1">
+        <header class="header">
+            <h3>Tapish Khari</h3>
+            <nav><i class="fa-solid fa-bars"></i></nav>
+        </header>
+        <div class="section">
+            <h1>Hey There.
+                I’m Tapish</h1>
+            <h2>I am a Software Developer</h2>
```

Comment:
- [ ] Please use `p` tag instead of `h2` for this part "I'm a software ..."   because it's part of the requirement and this way will be semantically correct and the screen readers :+1: